alb-ingress-controller-helm/values.yaml

```diff
@@ -17,9 +17,10 @@ clusterName: k8s
 controller:
   image:
     repository: quay.io/coreos/alb-ingress-controller
-    tag: "1.0-beta.4"
+    tag: "1.0-beta.5"
```

Comment:
This tag doesn't exist in the repo yet. `1.0-beta.4` is the latest.

https://quay.io/repository/coreos/alb-ingress-controller?tab=tags and filter `1.0`